apps/api/src/app/blueprint/usecases/get-grouped-blueprints/get-grouped-blueprints.usecase.ts

```diff
@@ -52,12 +52,12 @@ export class GetGroupedBlueprints {
   ): INotificationTemplate[] {
     const storedBlueprints = this.extractStoredBlueprints(groups);
 
-    const localPopularBlueprints = { ...POPULAR_TEMPLATES_GROUPED };
+    const localPopularIds = [...POPULAR_TEMPLATES_ID_LIST];
 
-    return localPopularBlueprints.blueprints.map((localBlueprint) => {
-      const storedBlueprint = storedBlueprints.find((blueprint) => blueprint._id === localBlueprint._id);
+    return localPopularIds.map((localBlueprintId) => {
+      const storedBlueprint = storedBlueprints.find((blueprint) => blueprint._id === localBlueprintId);
 
-      return (storedBlueprint ? storedBlueprint : localBlueprint) as INotificationTemplate;
+      return (storedBlueprint ? storedBlueprint : localBlueprintId) as INotificationTemplate;
```

Comment:
I don't follow on this one. It seems it either returns a blueprint (storedBlueprint which is type INotificationTemplate) or just an id (localBlueprintId which is string), but then cast to INotificationTemplate.
What am I missing?

Reply:
Totally right missed the update here, let me do quick fix here 🫠